tinygrad/tensor.py

```diff
@@ -36,7 +36,10 @@ def __exit__(self, *junk):
 def require_init_gpu():
   global cl_ctx, cl_queue
   if cl_queue is None:
-    cl_ctx = cl.create_some_context(interactive=False)
+    devices = cl.get_platforms()[0].get_devices(device_type=cl.device_type.GPU)
+    if len(devices) == 0:
+      devices = cl.get_platforms()[0].get_devices(device_type.cl.device_type.CPU)
```

Comment:
There's a typo here. "." -> "="

Reply:
@geohot Yes my bad, for some reason didn't think to test it beforehand... Anyways now it works and tests pass! :)